### PR TITLE
doc/blog: atlas wilco quest announcement

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/website/blog/2022-12-20-wilco-atlas-quest.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+technology. In order to truly understand how a tool works, you need to use it and see how it behaves in different
+situations.
+
+This is why I'm excited to announce that we've partnered with [Wilco](https://trywilco.com/) to create a new
+kind of learning experience for Atlas users. Wilco develops a platform, built to emulate the conditions at a tech
+startup, sends users on "quests" that cover everyday engineering tasks - from deploying an app to finding the root cause
+of a production issue - utilizing real-life tech stacks.
+
+When I first heard about Wilco, I was immediately intrigued by the idea of using a simulation game to
+teach people about database migrations and especially about solving all kinds of issues that many engineers
```
Comment (hilakashai) [inline doc/website/blog/2022-12-20-wilco-atlas-quest.md:21]:
```suggestion
teach people about database migrations. This platforms allows engineers to learn how to solve all kinds of issues that many 
```

File: doc/website/blog/2022-12-20-wilco-atlas-quest.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+only hit during a serious outage when there's very little room for error.
+
+### Introducing Wilco
+
+When you use Wilco, you're put in the shoes of a software engineer at a fictional startup. You're given access
+to a made up chat system humoristically called "Snack" where you meet different characters that you can interact with.
+You're also given access to a real GitHub repo and the means to run a real development environment on your local
```
Comment (hilakashai) [inline doc/website/blog/2022-12-20-wilco-atlas-quest.md:28]:
```suggestion
You are also given access to a real GitHub repo and the means to run a real development environment on your local
```

**Review by hilakashai — COMMENTED** [review]
Beast

File: doc/website/blog/2022-12-20-wilco-atlas-quest.md (hunk 1 above)
Comment (rotemtam) [reply doc/website/blog/2022-12-20-wilco-atlas-quest.md:21]: I will split into two but the stress here isn't on the platform but on the opportunity to practice "disastrous scenarios"

File: doc/website/blog/2022-12-20-wilco-atlas-quest.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+authors: rotemtam
+tags: [simulation, learning, wilco]
+image: https://atlasgo.io/uploads/images/explore_twitter_card.png
+---
+
+### Learning new things 
+
+One of my favorite things about software engineering, is that it's a career of continuous learning.
+Every new project or challenge presents us with an opportunity to increase our knowledge and improve our skills.
+Different people learn in different ways, but for me, one of the most effective ways to learn things has always been by
```
Comment (hilakashai) [inline doc/website/blog/2022-12-20-wilco-atlas-quest.md:12]:
```suggestion
Different people learn in different ways, but for me, one of the most effective ways to learn something new has always been by
```

File: doc/website/blog/2022-12-20-wilco-atlas-quest.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+doing. Sure, reading books, watching video and immersing myself in technical documentation is foundational, but
```
Comment (hilakashai) [inline doc/website/blog/2022-12-20-wilco-atlas-quest.md:13]:
```suggestion
doing. Sure, reading books, watching videos and immersing myself in technical documentation is foundational, but
```

File: doc/website/blog/2022-12-20-wilco-atlas-quest.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+I only feel like I really understand something is after I've applied it in some practical way.
```
Comment (hilakashai) [inline doc/website/blog/2022-12-20-wilco-atlas-quest.md:14]:
```suggestion
I only feel like I really understand something after I've applied it in some practical way.
```

File: doc/website/blog/2022-12-20-wilco-atlas-quest.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+This is why I'm excited to announce that we've partnered with [Wilco](https://trywilco.com/) to create a new
+kind of learning experience for Atlas users. Wilco develops a platform, built to emulate the conditions at a tech
+startup, sends users on "quests" that cover everyday engineering tasks - from deploying an app to finding the root cause
+of a production issue - utilizing real-life tech stacks.
+
+When I first heard about Wilco, I was immediately intrigued by the idea of using a simulation game to
+teach people about database migrations. Particularly interesting is the possibility to present people with
```
Comment (hilakashai) [inline doc/website/blog/2022-12-20-wilco-atlas-quest.md:22]:
```suggestion
teach people about database migrations. Particularly interesting, is the possibility to present people with
```

**Review by hilakashai — APPROVED** [review]

**Pull request merged**